onnxsim/onnx_simplifier.py

```diff
@@ -111,6 +111,73 @@ def get_input_names(model: onnx.ModelProto) -> List[str]:
     return input_names
 
 
+def get_elem_size_from_elem_type(elem_type: int):
+    return np.dtype(get_np_type_from_elem_type(elem_type)).itemsize
+
+
+def get_model_info(model: onnx.ModelProto):
+    """
+    op_dict
+    weight_num
+    """
+    op_dict = dict()
+    for node in model.graph.node:
+        if node.op_type not in op_dict.keys():
+            op_dict[node.op_type] = 1
+        else:
+            op_dict[node.op_type] = op_dict[node.op_type] + 1
+    weight_num = 0
+    for init in model.graph.initializer:
+        num = 1
+        if len(init.dims) != 0:
+            for dim in init.dims:
+                num *= dim
+        else:
+            size_bytes = len(init.raw_data)
+            elem_size = get_elem_size_from_elem_type(init.data_type)
+            num = size_bytes / elem_size
+        weight_num += int(num)
+    model_info = dict()
+    model_info.update({"op_dict": op_dict})
+    model_info.update({"weight_num": weight_num})
+    return model_info
+
+
+def print_sim_model_info(model_ori: onnx.ModelProto, model_opt: onnx.ModelProto):
+    """
+    --------------------------------------------
+    | type           | num of ori | num of opt |
+    --------------------------------------------
+    | ****           | ****       | ****       |
+    --------------------------------------------
+    | num of weight  | ***        | ***        |
+    --------------------------------------------
+    """
+    ori_info = get_model_info(model_ori)
+    opt_info = get_model_info(model_opt)
+    op_dict = dict()
+    for key, value in ori_info["op_dict"].items():
+        op_dict[key] = list()
+        op_dict[key].append(value)
+        op_dict[key].append(0)
+    for key, value in opt_info["op_dict"].items():
+        if key in op_dict:
+            op_dict[key][1] = value
+        else:
+            op_dict[key] = list()
+            op_dict[key].append(0)
+            op_dict[key].append(value)
+    str_format = "{0:^25}\t{1:^15}\t{2:^15}"
+    print("------------------------------------------------------------")
+    print(str_format.format("type", "model of ori", "model of sim"))
```

Comment:
Users does not know what is "ori". A more human-readable description will be better.